interview-questions.md

```diff
@@ -9,59 +9,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: From memory, this is a keyword in Javascript that is used to reference objects and access the values within objects/classes.
 
-  Researched answer:
+  Researched answer: After looking up this answer in the Syllabus, it appears I was on target with my initial answer.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: From memory, React is a Javscript framework used for creating user interfaces. One would use React if they want to quickly spin up a project and value scaleability.
 
-  Researched answer:
+  Researched answer: React was made to create user interfaces. It can be used to build web and mobile pages. Everything in React is a componenet. It is highly scaleable and built by Facebook.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: From memory, I believe the required lifecycle method in this case is the constructor.
 
-  Researched answer:
+  Researched answer: After looking up this answer in the syllabus, it appears I was on target with my initial answer.
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: From memory, JSX is the marriage of javascript and html. JSX is always wrapped in { }
 
-  Researched answer:
+  Researched answer: JSX is JavaScript flavored HTML. JSX used the syntax className since class is reserved in HTML.
 
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: From memory, yarn is a modern package manager used for managing projects
 
-  Researched answer:
+  Researched answer: I believe that after researching this one, I am happy with my initial answer.
 
 
 
 6. STRETCH: What is an anonymous function in JavaScript?
 
-  Your answer:
+  Your answer: If I remember correctly, an anonymous function is a function without a name.
 
-  Researched answer:
+  Researched answer: After researching this answer, it appears I was on target in my initial answer.
```

Comment:
Good job in this section.  I would recommend seeing if you can expand even further in your researched answers.